package.json

```diff
@@ -79,6 +88,7 @@
     "remark-preset-lint-node": "^3.4.0",
     "remark-preset-prettier": "^2.0.1",
     "sharp": "^0.31.3",
+    "storybook": "^7.0.0-rc.8",
```

Comment:
Just curious as to why we are using the release candidate? The latest version atm seems to be 6.5.16

Reply:
Looks like Storybook's own guides (and CLI tooling for setting up Storybook) chose this version.

Reply:
We can probably use it here as well if this RC is going to be released soon, will save troubles of migration